packages/utils/CHANGELOG.md

```
@@ -5,6 +5,17 @@
 
 ---
 
+## 5.0.0+
+
+### ⚠ BREAKING CHANGES
+
+* drop Node.js < 22.18.0 support
+* only support egg@4
+
+part of https://github.com/eggjs/egg/issues/5434
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)